src/xclim/indices/_agro.py

```diff
@@ -171,122 +179,89 @@ def huglin_index(
+    :math:`k`, based on latitude, :math:`lat`, is also considered. Then the heliothermal index for dates between
     1 April and 30 September is:
 
     .. math::
 
-       HI = \sum_{i=\text{April 1}}^{\text{September 30}} \left( \frac{TX_i  + TG_i)}{2} - T_{thresh} \right) * k
-
-    For the `smoothed` method, the day-length multiplication factor, :math:`k`, is calculated as follows:
+       HI = \sum_{i=\text{April 1}}^{\text{September 30}} \left(\frac{TX_i + TG_i)}{2} - T_{thresh} \right) * k
 
-    .. math::
-
-       k = f(lat) = \begin{cases}
-                     1, & \text{if } | lat | <= 40 \\
-                     1 + ((abs(lat) - 40) / 10) * 0.06, & \text{if } 40 < | lat | <= 50 \\
-                     NaN, & \text{if } | lat | > 50 \\
-                     \end{cases}
+    There are a few methods provided for calculating the day-length multiplication factor (:math:`k`) based on latitude:
 
-    For compatibility with ICCLIM, `end_date` should be set to `11-01`, `method` should be set to `icclim`.
-    The day-length multiplication factor, :math:`k`, is calculated as follows:
+    - For the `"huglin"/"icclim"` and `"interpolated"` methods, values for k increase from `1.0` at 40°N or 40°S to `1.06` at 50°N or 50°S,
+      where the `interpolated` method uses a smoothed curve and the `huglin/icclim` method uses a stepwise function.
+      Values above 50°N or below 50°S are set via the `cap_value` variable, with `1.0` set as default.
+      See: :py:func:`xclim.indices.helpers.day_length_latitude_coefficient` for more information.
+    - For the `"jones"` method, A more robust day-length calculation based on latitude, calendar, day-of-year,
+      and obliquity is used. The current implementation requires an annual frequency for consistent results.
+      See: :py:func:`xclim.indices.generic.jones_day_length_coefficient` or :cite:t:`hall_spatial_2010` for more information.
 
-    .. math::
-
-       k = f(lat) = \begin{cases}
-                     1.0, & \text{if } | lat | <= 40 \\
-                     1.02, & \text{if } 40 < | lat | <= 42 \\
-                     1.03, & \text{if } 42 < | lat | <= 44 \\
-                     1.04, & \text{if } 44 < | lat | <= 46 \\
-                     1.05, & \text{if } 46 < | lat | <= 48 \\
-                     1.06, & \text{if } 48 < | lat | <= 50 \\
-                     NaN, & \text{if } | lat | > 50 \\
-                     \end{cases}
-
-    A more robust day-length calculation based on latitude, calendar, day-of-year, and obliquity is available with
-    `method="jones"`. See: :py:func:`xclim.indices.generic.day_lengths` or :cite:t:`hall_spatial_2010` for more
-    information.
+    For compatibility with the original ICCLIM implementation :cite:p:`project_team_eca&d_algorithm_2013`,
+    `end_date` should be set to `11-01` with `method="huglin"` or `method="icclim"` (deprecated).
```

Comment:
I'm in favor of removing icclim for the actual name of the method. I'm a bit on the fence on this one. Maybe it's better to add a note to state that hughlin corresponds to icclim in old xclim versions. BUT, we do breaking changes from time to time, and we simply trust that our changelog is sufficient to inform users. So maybe it's not necessary actually